src/encoded/static/components/publication.js

```diff
@@ -45,6 +104,21 @@ const PublicationComponent = (props, reactContext) => {
                 </div>
             : null}
 
+            {context.datasets && context.datasets.length ?
+                <SortTablePanel title="Datasets">
+                    <FetchedData ignoreErrors>
+                        <Param name="datasets" url={datasetsUrl} />
+                        <DatasetsComponent columns={datasetsColumns} />
+                    </FetchedData>
+                </SortTablePanel>
+            : null }
+
+            {context.datasets && context.datasets.length ?
```

Comment:
What is this section for? It looks like it would do a GET request for an Experiment search, and then throw away the result. Does the result get directed somewhere else I’m not aware of?

Reply:
Based on looking at it, it may have been me experimenting with SortTable prior to implementing this feature. I removed it as it has no use now.